### PR TITLE
Feature/line chart

### PR DESCRIPTION
**Pull request by behzadiv**

### CODE REVIEW

File: src/components/coinPriceChange/content/CoinPriceChange.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { CoinPriceChangeProps } from "./coinPriceChange_types";
+
+export const CoinPriceChangeSection: React.FC<CoinPriceChangeProps> = ({
+  priceChange,
+}) => {
+  if (priceChange === 0)
+    return (
+      <PriceChangeContainer priceChange={priceChange}>0</PriceChangeContainer>
+    );
+  return priceChange < 0 ? (
```
Comment (alirezatahani0) [inline src/components/coinPriceChange/content/CoinPriceChange.tsx]: no duplicate code is allowed

File: src/components/coinsList/content/CoinList.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          return (
+            <CoinItem
+              key={uuid}
+              uuid={uuid}
+              change={Number(change)}
+              iconUrl={iconUrl}
+              name={name}
+              price={price}
+              sparkline={sparkline}
+            />
```
Comment (alirezatahani0) [inline src/components/coinsList/content/CoinList.tsx]: you dont need to pass each of them with same key and value just use spread syntax

**Review by alirezatahani0 — CHANGES_REQUESTED** [review]

File: src/components/coinPriceChange/content/CoinPriceChange.tsx (hunk 1 above)
Comment (behzadiv) [reply src/components/coinPriceChange/content/CoinPriceChange.tsx]: resolved , although we removed this section after littleLineChart added instead

File: src/components/coinsList/content/CoinList.tsx (hunk 2 above)
Comment (behzadiv) [reply src/components/coinsList/content/CoinList.tsx]: resolved

**Pull request merged**